base/expr.jl

```diff
@@ -109,13 +109,17 @@ evaluates expressions in that module.
 Core.eval
 
 """
-    @eval
+    @eval [m,] ex
 
-Evaluate an expression and return the value.
+Evaluate an expression with values interpolated into it using `eval`.
+If two arguments are provided, the first is the module to evaluate in.
 """
 macro eval(x)
     :($(esc(:eval))($(Expr(:quote,x))))
 end
+macro eval(m, x)
+    :($(esc(:eval))($(esc(m)),$(Expr(:quote,x))))
```

Comment:
Probably true, but I think that's a bugfix rather than a feature addition, so separate PR.